src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.cs

```diff
@@ -144,8 +145,10 @@ internal static async Task<IEnumerable<Diagnostic>> GetDiagnosticsAsync(SyntaxTr
 
             var message = string.Format(
                 EditorFeaturesResources.Rename_0_to_1,
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
not sure what are obsolete.

Reply:
Thanks, I forgot to remove these lines when I removed `[Obsolete]` from `RenameTrackingDiagnosticAnalyzer`.